### PR TITLE
add npow chain apis

### PR DESCRIPTION
**Pull request by xcaptain**

### CODE REVIEW

**Comment by xcaptain** [comment]
Need some advice about naming, who to maintain whitelist ...

**Comment by xcaptain** [comment]
see also: https://github.com/deeper-chain/dpr-ezc-bridge/pull/4

File: pallets/operation/src/lib.rs [hunk 1]
```diff
@@ -428,6 +430,32 @@ pub mod pallet {
...
         }
+
+        // TODO: use real weight
+        #[pallet::weight(T::OPWeightInfo::start_withdraw_ezc())]
+        pub fn start_withdraw_ezc(
+            origin: OriginFor<T>,
+            eth_address: H160,
+        ) -> DispatchResultWithPostInfo {
+            let sender = ensure_signed(origin)?;
+            Self::deposit_event(Event::<T>::StartWithdrawEZC(sender, eth_address));
```
Comment (jerry-yu) [inline pallets/operation/src/lib.rs:441]: only send event ?

Comment (xcaptain) [reply pallets/operation/src/lib.rs]: Yes, `get_npow_reward` send a event, bridge listen this event, invoke the contract to compute its npow reward, then call `npow_mint` to give some tokens to the account.

**Comment by xcaptain** [comment]
@jerry-yu @AmadeusGB please look again, are you ok with the naming and whitelist implementation.

**Review by jerry-yu — APPROVED** [review]

**Pull request merged**